bin/zf-maintainer.php

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env php
+<?php
```

Comment:
Add a file-level docblock indicating how the script is invoked (essentially the usage information). This docblock should also include the license, copyright, etc. we normally have.